paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -318,7 +317,6 @@ internal abstract class BaseSheetViewModel(
             paymentMethodsValue != null &&
             isGooglePayReadyValue != GooglePayState.Indeterminate &&
             isResourceRepositoryReadyValue &&
-            isLinkReadyValue != null &&
```

Comment:
Does this change any behavior? Previously, `isLinkReadyValue` was null until it was set. Now, it’s `false` initially.

Reply:
This PR doesn't change behavior, but https://github.com/stripe/stripe-android/pull/6004 did.

This looks like a potential breaking change. So I'm going to remove that from this PR and fix that independently in another PR.